index.js

```diff
@@ -144,18 +144,28 @@ const matchesRules = (ruleset, msg) => Object.keys(ruleset).some(key => {
 	return !!msg[key] && ruleset[key].test(msg[key]);
 });
 
+const getFirstNode = (nodeOrList) => {
+	return nodeOrList instanceof global.HTMLElement ? nodeOrList : nodeOrList[0];
+};
+
 const resolveAttribute = (name) => (node) => {
-	if (!node) return null;
+	if (!node || node.length === 0) return null;
+
+	if (name === '<count>') {
```

Comment:
Is there someplace to document this special attribute?

Reply:
Working on it now